src/open_inwoner/cms/cases/views/status.py

```diff
@@ -414,11 +414,26 @@ class CaseContactFormView(CaseAccessMixin, CaseLogMixin, FormView):
 
     def post(self, request, *args, **kwargs):
         form = self.get_form()
+
         if form.is_valid():
-            form.cleaned_data[
-                "question"
-            ] += f"\n\nCase number: {self.case.identificatie}"
-            return self.register_by_api(form)
+            config = OpenKlantConfig.get_solo()
+
+            success = True
+            if config.register_email:
+                form.cleaned_data[
+                    "question"
+                ] += f"\n\nCase number: {self.case.identificatie}"
+                success = (
+                    self.register_by_email(form, config.register_email) and success
```

Comment:
Is `and success` necessary here, seeing that it is set to `True` at L421?

Reply:
We used this with Bart and if I remember right we did it for readability purposes. The variable is modified a lot of times so it is better for the user to understand what's going on.